Sources/StreamChat/Workers/MessageUpdater_Tests.swift

```diff
@@ -988,9 +988,7 @@ final class MessageUpdater_Tests: XCTestCase {
         // wait for the db call to be done
         wait(for: [dbCall], timeout: 0.1)
 
-        let reaction = database.viewContext.reaction(messageId: messageId, userId: userId, type: reactionType)
-
-        guard let reaction = reaction else {
+        guard let reaction = database.viewContext.reaction(messageId: messageId, userId: userId, type: reactionType) else {
```

Comment:
This is just a nitpick.

In cases like these, for readability/maintenance reasons we tend to break params on the newline.

```suggestion
        guard let reaction = database.viewContext.reaction(messageId: messageId, 
                                                                                               userId: userId, 
                                                                                               type: reactionType) else {
```

Reply:
We set our formatter line length to 132 and it does this automatically, the line doesn't exceed the limit